visualization/app/codeCharta/codeCharta.mouseEvent.service.ts

```diff
@@ -6,6 +6,7 @@ import { setSearchPanelMode } from "./state/store/appSettings/searchPanelMode/se
 export class CodeChartaMouseEventService {
 	constructor(private storeService: StoreService) {}
 
+	// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Would be great to have a TODO here as well. I'm afraid we miss this in the future otherwise.

Reply:
Done